protonj2-client/src/test/java/org/apache/qpid/protonj2/client/impl/ReceiverTest.java

```diff
@@ -1305,15 +1305,16 @@ public void doTtestBlockingReceiveCancelledWhenReceiverClosedOrDetached(boolean
 
     @Test
     public void testBlockingReceiveCancelledWhenReceiverRemotelyClosed() throws Exception {
-        doTtestBlockingReceiveCancelledWhenReceiverClosedOrDetached(true);
+        doTestBlockingReceiveCancelledWhenReceiverClosedOrDetached(true);
     }
 
     @Test
     public void testBlockingReceiveCancelledWhenReceiverRemotelyDetached() throws Exception {
-        doTtestBlockingReceiveCancelledWhenReceiverClosedOrDetached(false);
+        doTestBlockingReceiveCancelledWhenReceiverClosedOrDetached(false);
     }
 
-    public void doTtestBlockingReceiveCancelledWhenReceiverRemotelyClosedOrDetached(boolean close) throws Exception {
+    // TODO unused
+    public void doTestBlockingReceiveCancelledWhenReceiverRemotelyClosedOrDetached(boolean close) throws Exception {
```

Comment:
I'll do a fixup commit, the two tests above are obviously calling a wrong do method.